packages/sign-client/package.json

```diff
@@ -26,6 +26,7 @@
     "build": "run-s build:pre build:cjs build:esm build:umd",
     "test:pre": "rm -rf ./test/test.db",
     "test:run": "env TS_NODE_PROJECT=\"tsconfig.cjs.json\" mocha --timeout 10000 --exit --r ts-node/register ./test/**/*.spec.ts",
+    "test:concurrency": "env TS_NODE_PROJECT=\"tsconfig.cjs.json\" mocha --timeout 100000 --exit --r ts-node/register ./test/concurrency.spec.ts",
```

Comment:
nit: currently `test:` is used internally ony.

I suggest `npm run loadtest` instead